Google.Api.Generator.Tests/ProtoTests/Basic/BasicClientSnippets.g.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System.Threading.Tasks;
+
+namespace Testing.Basic
+{
+    /// <summary>Generated snippets.</summary>
+    public sealed class GeneratedBasicSnippets
+    {
+        /// <summary>Snippet for IdempotentMethod</summary>
+        public void IdempotentMethod_RequestObject()
+        {
+            // Snippet: IdempotentMethod(Request, CallSettings)
+            // Create client
+            BasicClient basicClient = BasicClient.Create();
+            // Initialize request argument(s)
```

Comment:
I hope that eventually we'll be able to get rid of the "(s)" bit. It bugs me :)
(Not suggesting a change now though.)

Reply:
Agreed :)